### PR TITLE
Added protecc init

### LINKED ISSUE #5: init command
Add an init command that puts a .pysa and/or tainted file to the user's working directory

### PR DESCRIPTION
**Pull request by Saif807380**
The `protecc init` command generates a `models.pysa` file and a `taint.config` file. Currently, both files are empty. The command also takes an option `--config-directory` where the user can provide the path where the config files must be created. Fixes #5.

### Setup instructions

1. Inside your virtual env run the below command from the project root. This will install the `protecc` command

```bash
$ pip install --editable .
```

2. Run the command

```bash
$ protecc init --config-path="./src/config"
```

### Changes made 

- Created a new file `commands.py` that will contain all the `protecc` commands.
- Added `setup.py` file for running the commands.
- Added option for providing a path where the config files should be generated.

### Screenshots

Screenshot 2021-02-04 at 12 11 04

### CODE REVIEW

File: src/commands.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import os
+
+@click.group()
+def protecc():
+    pass
+
+@protecc.command()
+def hello():
+    """Example script."""
+    click.echo('Hello World!')
```
Comment (SaurusXI) [inline src/commands.py]: I think we should omit this

**Comment by SaurusXI** [comment]
Looks good! It's missing the template `.pysa` and `taint.config` files right now. We'll have to add that too

File: src/commands.py (hunk 1 above)
Comment (Saif807380) [reply src/commands.py]: Yup. I'll remove it with the next commit.

**Comment by Saif807380** [comment]
Let's do one thing. Let's put merging this on hold till we add the content of the files.

**Comment by SaurusXI** [comment]
Sorry for all the additional commits, I did a rebase :sweat_smile: 
This should work, but we'll know for sure after integration testing. I think we can get on that as soon this is merged

**Review by gasharova — APPROVED** [review]
LGTM. Real deal will be tomorrow when we test how things integrate with each other.

**Pull request merged**